src/ansys/materials/manager/util/matml/matml_from_material.py

```diff
@@ -193,11 +179,47 @@ def export(self, path: _PATH_TYPE, **kwargs):
 
         # add transfer id to the XML tree
         self._add_transfer_ids(root)
+        return tree
+
+    def _indent(self, tree) -> None:
+        if hasattr(ET, "indent"):
+            ET.indent(tree)
+        else:
+            print(f"ElementTree does not have `indent`. Python 3.9+ required!")
+
+    def write(self, buffer: BinaryIO, **kwargs) -> None:
+        """
+        Write a Matml (engineering data xml format) representation of materials to buffer.
+
+        Parameters
+        ----------
+        buffer:
+            buffer to write to
+        Keyword arguments
+        -----------------
+        indent - add an indent to format the XML output (python 3.9+). Defaults to false.
+        """
+        tree = self._to_etree()
+
+        if kwargs.get("indent", False):
```

Comment:
Can you parameterize the read/write test with indent as a bool to cover these two missing hunks?